testsuite/features/support/api_test.rb

```diff
@@ -0,0 +1,587 @@
+# Copyright (c) 2011-2022 SUSE LLC.
```

Comment:
WOW!
I just noticed something that I totally dislike!
We have move our organized and small classes to handle XMLRPC calls by namespace.
And we now have all at same file :(

We have now another file with more than 500 lines, I think this is really not a good practice.

Reply:
My suggestion is to have an APIHandler, following a composition pattern, which handle all the multiple classes per API namespace.
It will expose these attributes or we can use getters, as we prefer, so we can have something like `api.my_namespace.my_method(...)`
We can also share the connection, by sharing something like `@connection` when we initialize the different classes, so it will login when APIHandler initiate and then share the connection.

Reply:
After discussion in slack, fixed.